src/errors.h

```diff
@@ -47,6 +47,5 @@
 --IO500 - GROUP: internal error
 --IO501 - unexpected state/event
 */
-#define ERRCODE_TS_INTERNAL_ERROR MAKE_SQLSTATE('T', 'S', '5', '0', '0')
```

Comment:
This is barely used and of unclear utility, so I replaced it with the normal Postgres code "ERRCODE_INTERNAL_ERROR".